x-pack/legacy/plugins/apm/public/components/shared/TransactionActionMenu/CustomLink/CustomLinkSection.tsx

```diff
@@ -19,7 +31,7 @@ export const CustomLinkSection = ({
   customLinks: CustomLink[];
   transaction: Transaction;
 }) => (
-  <SectionLinks>
+  <ul>
```

Comment:
Should this change not be made in the shared `SectionLinks` component?

Reply:
I agreed, but due to the deadline of the release I thought to do it here, and create a new issue to replace it later on `SectionLinks`, WDYT?